<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>412</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QPushButton" name="tlacitko_start">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QWidget" name="obalovac" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="6" column="3">
        <widget class="QPushButton" name="tlacitko_tak_to_ukoncime">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Konec zavodu</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QCheckBox" name="zvladam_cist_popisky">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Zvladam cist popisky</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLabel" name="casoukazovac_celkovy">
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>00:00:00</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="casoukazovac_relativni">
         <property name="font">
          <font>
           <family>DejaVu Sans Mono</family>
          </font>
         </property>
         <property name="text">
          <string>00:00:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="tlacitko_vyexportuj_to">
         <property name="text">
          <string>Vyexportuj to</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="4">
        <widget class="QTableView" name="tabulkohled">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QPushButton" name="tlacitko_stopni_to">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Nekdo probehl cilem</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuManu">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuV">
     <property name="title">
      <string>V</string>
     </property>
     <widget class="QMenu" name="menuY">
      <property name="title">
       <string>y</string>
      </property>
      <widget class="QMenu" name="menuP">
       <property name="title">
        <string>p</string>
       </property>
       <widget class="QMenu" name="menuN">
        <property name="title">
         <string>n</string>
        </property>
        <widget class="QMenu" name="menuO">
         <property name="title">
          <string>o</string>
         </property>
         <widget class="QMenu" name="menuU">
          <property name="title">
           <string>u</string>
          </property>
          <addaction name="actionT"/>
         </widget>
         <addaction name="menuU"/>
        </widget>
        <addaction name="menuO"/>
       </widget>
       <addaction name="menuN"/>
      </widget>
      <addaction name="menuP"/>
     </widget>
     <addaction name="menuY"/>
    </widget>
    <addaction name="menuV"/>
   </widget>
   <addaction name="menuManu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionT">
   <property name="text">
    <string>t</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
